skysurvey/target/kilonova.py

```diff
@@ -1,12 +1,32 @@
 from .core import Transient
-
+#This is the code to model kilonova
+import sncosmo as sc
+from simsurvey.utils import model_tools
+from simsurvey.models import AngularTimeSeriesSource
+import numpy as np
+optional_injection_parameters={}
+phase, wave, cos_theta, flux = model_tools.read_possis_file(
```

Comment:
simsurvey should not be a dependency of skysurvey. You should copy in what code you need to read this file. You will also need to add the file into the repo in the same pull request.

Reply:
ok will do that